build.gradle

```diff
@@ -73,9 +73,16 @@ ext {
     log4j: "log4j:log4j:1.2.17",
     objenesis: "org.objenesis:objenesis:3.0.1"
   ]
-  Date buildTimeAndDate = new Date()
-  buildDate = new SimpleDateFormat('yyyy-MM-dd').format(buildTimeAndDate)
-  buildTime = new SimpleDateFormat('HH:mm:ss.SSSZ').format(buildTimeAndDate)
+  if (snapshotVersion && System.env.CI != "true") {
+    //build time is not crucial for local SNAPSHOT builds, unless published from CI server
+    //speed up local development by leveraging Gradle caching mechanism
+    buildDate = "2020-02-02"
```

Comment:
Some reproducible build tools set it to the begin of unix-time (1970-01-01 00:00:00.000Z), I think that would be a better default.

Reply:
I know, but 20200202 is also a nice date :).

Changed to the Unix epoch.